### ISSUE TITLE
BlueMSX core is running but not displaying anything on macOS

### ISSUE THREAD

**Issue by fmaida** [libretro/ludo#143]
Hello, I have a Mac with macOS 10.14.2 (Mojave) and with an Intel HD Graphics 5000 with 1,5Gb of VRAM.  

Premise
=======
All the libretro cores I'm using have been downloaded by using RetroArch and its "Online updater" function; Then I have just changed the settings in Ludo to point the core folder to the same core folder of RetroArch.

I'm trying to play some old MSX games by using the BlueMSX core, and when I try to launch a game under RetroArch with BlueMSX core, I can see that it works properly. 

My issue with Ludo
==============

When I try to launch the same MSX game under Ludo, even if I can notice that BlueMSX core is running underneath and I can hear audio unfortunately I cannot see anything but a black screen. If I try to exit from the game and go back into ludo, I can do it successfully usually just once. The third time I try to load a MSX game, ludo crashes. Colecovision games are giving me the same behaviour and, if I'm not mistaken, they're also using BlueMSX core; Other emulation cores I've tried are working properly AFAIK.
I'm having this problem with both Ludo 0.3.1 and 0.3.2. 

I'm attaching a partial copy of my console log: [ludo.log](https://github.com/libretro/ludo/files/2829754/ludo.log)

Video recording
============

I've also recorded this strange behaviour on video, here is the link to a unlisted YouTube video I've just taken:

https://youtu.be/K9rPOhuvWkc

P.S. : BTW Great work with Ludo. I'm sure that once you'll publish it officially, everyone will love it.

**Comment by kivutar** [comment]
I'm also experiencing this bug. It may have to do with a `pixFmt` or `pixType` that Ludo doesn't support yet.
Or something with the framebuffer pitch.

**Comment by kivutar** [comment]
I've fixes something that may help here. However, I'm not able to launch an MSX rom anymore. I'm getting a `failed to load the game` from the LoadGame call.

**Comment by mgmalheiros** [comment]
Trying the same here. I was able to pass the `failed to load the game` message by adding the **Databases** and **Machines** folders (with related contents), as described in

  https://github.com/libretro/blueMSX-libretro

MSX machines need BIOS roms, and there is a opensource BIOS included with blueMSX.

I actually got the `[Core]: Game loaded` message on console, but after that Ludo crashes. As I am using Ludo 0.6.1 on Ubuntu 14.04 LTS with a recent blueMSX retroarch core, the crash is probabbly caused by my old runtime libraries on Linux side.
